contents/tutorials/session-recordings-for-support.md

```diff
@@ -0,0 +1,116 @@
+---
+title: How to use session recordings to improve your support experience
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-12-22
+featuredImage: ../images/tutorials/banners/session-recordings-for-support.png
+topics: ['session recordings', 'sentry']
+---
+
+On top of being useful for understanding users’ behavior in your product, session recordings can help you solve problems with your product. You can use them to discover issues, understand why they are happening, and work to fix them. Using session recordings helps you provide a better support experience to users, and getting the most out of them is what we’ll go over in this tutorial.
+
+## Finding relevant sessions for users needing support
+
+When a user reports an issue, a session recording turns a long conversation into a short recording viewing. This is because you can see what actions they took, rather than asking them to explain their situation. It also provides easy access to relevant data and properties (like OS, browser, and custom properties). This helps your team diagnose issues and find solutions faster.
+
+To find recordings from a specific user needing support, you can search for the user in PostHog and check their session recordings tab. You can also filter the sessions recording page for specific users or events. For example, to find a session recording where I was having a problem, I can filter for the Rageclick event, email is mine, and within the last 24 hours.
```

Comment:
```suggestion
To find recordings from a specific user that needs support, you can search for them in PostHog's 'Persons & Groups' section, and check their session recordings tab. Alternatively, you can filter for a user or a recent event via the Sessions Recording page. For example, to find a recent session recording where I had a problem, I could filter for sessions from the last 24 hours which feature the `Rageclick` event and my email address.
```

Maybe this is simpler wording? Trying to avoid `Email is mine`, though the supporting image means it isn't a big issue either way. 🤷

Reply:
I'll accept your suggestion and work on it a bit.